docs/docs/development/new-shield.md

```diff
@@ -78,7 +80,7 @@ which controls the display name of the device over USB and BLE.
 The updated new default values should always be wrapped inside a conditional on the shield config name defined in the `Kconfig.shield` file. Here's the simplest example file.
 
 :::warning
-Do not make the keyboard name too long, otherwise the bluetooth advertising might fail and you will not be able to find your keyboard from your laptop / tablet.
+Do not make the keyboard name too long, otherwise the bluetooth advertising might fail and you will not be able to find your keyboard from your laptop / tablet. The recommended length is 16 characters or less.
```

Comment:
```suggestion
The keyboard name must be 16 characters or less. With longer names the bluetooth advertising might fail and you will not be able to find your keyboard from your laptop / tablet.
```

I found with a longer name the code won't compile on an assert. This is not a recommendation but a requirement.